### PR TITLE
Ford's Entando blog

### PR DESCRIPTION
**Pull request by fuhsaz**
Here's my blog from the bench project so far. It's about Personas, which are a framework Entando uses to describe the different roles people can play when using Entando to build modular applications. Currently the only documentation I could find from them was 4-5 videos, and I prefer to read things like that so hopefully other people do too.

### CODE REVIEW

**Review by cjin-1 — CHANGES_REQUESTED** [review]
maybe clarify on the hub that's mentioned throughout the blog - is it all talking about the Entando hub? or another? 

Otherwise, looks good to me!

File: posts/entando-personas-make-modular-applications-more-manageable.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+tags: 
+- Entando
+- Micro-frontends
+- Microservices
+date: 2023-01-12T9:15:00:00.000Z
+title: "Entando Personas Make Modular Applications More Manageable"
+image: ![Multiple people working on a web application](https://github.com/fuhsaz/blog-usa/blob/master/images/2023/01/entndo_personas_blog_image.jpg)
+---
+
+Over the past few years, it has become apparent that businesses need to adapt quickly to changing conditions to stay competitive. Pushing a new feature's time-to-market due to another application part's unpreparedness is unacceptable. Building with a modular architecture in mind can give development teams more independence and potentially improve overall throughput. Unfortunately, modularizing applications can quickly become very complicated and difficult to manage.
```
Comment (dsharpe) [inline posts/entando-personas-make-modular-applications-more-manageable.md:13]: The second sentence sounds a little awkward. Perhaps something like "...due to another part of the application being unprepared is unacceptable"

File: posts/entando-personas-make-modular-applications-more-manageable.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## What are Personas in Entando?
+Entando defines the Four C's of Application Composition as Create, Curate, Compose, and Consume. Each has a "Persona", or a role that one or more people can fill to focus their strengths towards where they can best contribute. Creators may be more comfortable working in a high-code/traditional-build environment, whereas Composers use a low-code interface to assemble the application. Curators manage and catalog components and identify what may still be needed, and Consumers populate the application with up-to-date and approved web content. 
+
+A single person or team may fit and act as more than one Persona at different times. Still, larger teams may wish to organize people by their technical or business specialties to parallelize the development process better.
```
Comment (dsharpe) [inline posts/entando-personas-make-modular-applications-more-manageable.md:23]:
```suggestion
A single person or team may fit and act as more than one Persona at different times. Still, larger teams may wish to organize people by their technical or business specialties to better parallelize the development process. 
```

File: posts/entando-personas-make-modular-applications-more-manageable.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+A single person or team may fit and act as more than one Persona at different times. Still, larger teams may wish to organize people by their technical or business specialties to parallelize the development process better. 
+
+## Creators
+Front-end and back-end developers likely find a Creator role similar to their experience. Creators are responsible for creating and packaging business capabilities, such as a micro-frontend and its associated microservice or a logo widget that can be reused in several of a company's applications, which can then be pushed to a central hub, which may be the official hub hosted by Entando or a internal hub shared throughout a company. A Creator can be a single developer, but there is room for design and business people in this role too! An entire development team consisting of design, development, testing, and business people can keep internally-created components consistent and aligned with business goals.
```
Comment (dsharpe) [inline posts/entando-personas-make-modular-applications-more-manageable.md:26]: The second sentence is a run-on sentence. I would suggest breaking it into 2 sentences or more.

File: posts/entando-personas-make-modular-applications-more-manageable.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Creators can quickly use the Entando Component Generator to generate the base code for new microservices and micro-frontends. The generator leverages [JHipster](https://www.jhipster.tech/) with [JHipster Blueprints](https://www.jhipster.tech/modules/extending-and-customizing/) and aims to accelerate development and standardize technology by allowing developers to quickly spin up new components with a consistent tech stack.
+
+There are a few things that Creators can keep in mind to allow the Application Composition process to flow as smoothly as possible:
+1. Loosely couple any components and fully map dependencies so that future developers can clearly understand any potential side effects when making changes.
+2. Document the custom web components used to inject micro-frontends into your parent application. It can be helpful to keep track of all attributes, custom events, and CSS properties these components may require to funciton.
+3. Take advantage of automation wherever possible. CI/CD pipelines save time, of course, but the Entando CLI has numerous features, such as boilerplate code generation and environment profiles, that can speed of development.
```
Comment (dsharpe) [inline posts/entando-personas-make-modular-applications-more-manageable.md:33]:
```suggestion
3. Take advantage of automation wherever possible. CI/CD pipelines save time, of course, but the Entando CLI has numerous features, such as boilerplate code generation and environment profiles, that can speed up development. 
```

File: posts/entando-personas-make-modular-applications-more-manageable.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+## Composers
+Composers work in the low-code environment of the Entando App Builder to combine the different business capabilities provided by the Creators and made available by the Curators to create an application that fits the business' needs. 
+
+A benefit of building your application in Entando is the Entando Hub, which contains numerous existing components that may fit your business case with little to no configuration required. The best code is often code your team does not need to write or maintain.
+
+Entando uses Role-Based Access Controls (RBAC), which restrict individual pages or directories  to people with the knowledge and business need to access them. Favoring components with this built-in functionality, or building components around RBAC from the beginning, will allow you to make sure that all users have the right level of access to all parts of your application without much extra work.
```
Comment (dsharpe) [inline posts/entando-personas-make-modular-applications-more-manageable.md:47]:
```suggestion
Entando uses Role-Based Access Controls (RBAC), which restrict individual pages or directories to people with the knowledge and business need to access them. Favoring components with this built-in functionality, or building components around RBAC from the beginning, will allow you to make sure that all users have the right level of access to all parts of your application without much extra work.
```

**Review by dsharpe — CHANGES_REQUESTED** [review]

File: posts/entando-personas-make-modular-applications-more-manageable.md (hunk 1 above)
Comment (fuhsaz) [reply posts/entando-personas-make-modular-applications-more-manageable.md:13]: I'll rewrite this and fix the other things you pointed out. Thanks for the review!

Comment (pkrish123) [reply posts/entando-personas-make-modular-applications-more-manageable.md:13]: I agree with Dennis! The second sentence is a little awkward and possibly a bit unclear. I would change the beginning of the sentence to "Delaying the release of a new feature" and then follow up with @dsharpe's amendment.

File: posts/entando-personas-make-modular-applications-more-manageable.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Consumers are the final step in the Application Composition process. They add and update web content within the page templates provided by the Composers, then set up analytics to track how that content performs. 
+
+Consumers are not necessarily restricted to managing content, however. They can create and manage users and roles, and edit rules and processes that run on the application. 
+Finally, Consumers can analyze application traffic and tune settings in the Entando Operator configuration to maximize performance.
+
+## In Conclusion
+With Personas, Entando provides a useful framework for every member of a team, depending on their technical specialties and business knowledge, to quickly find where they would be most comfortable and able to contribute. Creators build components, Curators validate and organize them, Composers combine them, and Consumers fill in the details, keep content up to date, and respond to feedback. 
+
+You can learn more about Entando by clicking [here](https://entando.com/), or more about JHipster [here.](https://www.jhipster.tech/) If you have any other questions, we would love to hear from you at [contact@ippon.tech](mailto:contact@ippon.tech). 
+
```
Comment (pkrish123) [inline posts/entando-personas-make-modular-applications-more-manageable.md:59]: This is really great @fuhsaz! I loved learning more about Entando's Personas over the course of this blog. Are there any diagrams that you can include to show the relationship between the 4 personas/ flow of development?

**Review by pkrish123 — CHANGES_REQUESTED** [review]

**Review by pkrish123 — APPROVED** [review]

**Review by dsharpe — APPROVED** [review]

**Pull request merged**